tests/integration/test_charm.py

```diff
@@ -23,6 +24,7 @@
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 APP_NAME = "seldon-controller-manager"
+SELDON_DEPLOYMENT = None
```

Comment:
This suggestion is related to [this comment](https://github.com/canonical/seldon-core-operator/pull/146/files#r1223350392)
```suggestion
SELDON_DEPLOYMENT = create_namespaced_resource(
    group="machinelearning.seldon.io",
    version="v1",
    kind="seldondeployment",
    plural="seldondeployments",
    verbs=None,
)
```

Reply:
+1